test/CMakeLists.txt

```diff
@@ -135,6 +134,32 @@ endif()
 compile_output_test(complexity_test)
 add_test(complexity_benchmark complexity_test --benchmark_min_time=${COMPLEXITY_MIN_TIME})
 
+###############################################################################
+# GoogleTest Unit Tests
+###############################################################################
+
+if (BENCHMARK_ENABLE_GTEST_TESTS)
+  include(HandleGTest)
+  include_directories(SYSTEM ${GTEST_INCLUDE_DIRS})
```

Comment:
```diff
- include_directories(SYSTEM ${GTEST_INCLUDE_DIRS})
```

Reply:
This line still needs to stay for when out-of-tree GTest installations are used. However the SYSTEM can be removed, since it seems to be implicitly specified after adding the `INTERFACE_SYSTEM_INCLUDE_DIRECTORIES` line.

Reply:
Nevermind... You're right.